electron/legendary/games.ts

```diff
@@ -659,15 +657,29 @@ class LegendaryGame extends Game {
       ]
     }
 
-    const { error, stderr, stdout, fullCommand } = await runLegendaryCommand(
+    const fullCommand = getRunnerCallWithoutCredentials(
       commandParts,
-      {
-        env: commandEnv,
-        wrappers: wrappers,
-        logMessagePrefix: `Launching ${gameInfo.title}`
-      }
+      commandEnv,
+      wrappers,
+      join(...Object.values(getLegendaryBin()))
+    )
+    appendFileSync(
+      this.logFileLocation,
+      `Launch Command: ${fullCommand}
+
+Game Log:
+`
```

Comment:
Same here. Maybe we think of a layout for games log aswell? Like general log has it? 
Calculated tabs/spaces etc.

Reply:
In terms of a "layout" for these logs: I'd say the current format is *fine* for what it needs to do. No need to over-complicate it